lib/services/firebase.dart

```diff
@@ -0,0 +1,50 @@
+// ignore_for_file: slash_for_doc_comments
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:lost_found_steelhacks/models/app_user.dart';
+
+/**
+ * Singleton service for making Firebase calls
+ */
+class FirebaseService {
```

Comment:
This seems wrong for some reason. Firebase itself is already a class and has methods in it, feels wrong to package it up again? Feel free to correct me if I'm wrong. Also what was with it being in the auth.dart file?

Reply:
Mostly just for the convenience of calling one method to do something. Take ```createUser``` as an example. It takes the ```AppUser``` datamodel and then performs the database call that creates the firebase mapping. I'm not opposed to moving these method(s) though to the ```AuthService``` class instead, that might make more sense - because i just realized that we have a service class calling a service class which is honestly probably not good.

Reply:
Your call on what to do about this, let me know, I can refactor it